swagger_to_sdk_config.json

```diff
@@ -1,16 +1,20 @@
 {
   "meta": {
     "after_scripts": [
-      "gofmt -w ./services/"
+      "gofmt -w ./services/",
+      "go get -u github.com/Azure/azure-sdk-for-go/tools/profileBuilder",
+      "profileBuilder -s list -l ./profiles/2017-03-09/defintion.txt -name 2017-03-09",
+      "profileBuilder -s preview -name preview",
+      "profileBuilder -s latest -name latest"
```

Comment:
for latest to respect the new folder structure of the repo we should use the list strategy and the stableApis.txt file from there

Reply:
Also are we looking to change the "latest" profile name before GA ? if so lets open an issue for that